isb_web/main.py

```diff
@@ -111,24 +113,36 @@ async def get_thing(
         raise fastapi.HTTPException(
             status_code=404, detail=f"Thing not found: {identifier}"
         )
-    if full:
+    if full or format == "full":
         return item
+    if format == "core":
+        authority_id = item.authority_id
+        if authority_id == "SESAR":
+            content = SESARTransformer(item.resolved_content).transform()
+        else:
+            raise fastapi.HTTPException(
+                status_code=404,
+                detail=f"Core format not available for authority_id: {authority_id}",
+            )
```

Comment:
Is raising an exception the right thing to do in this case?

Reply:
It's probably a 400 error, https://developer.mozilla.org/en-US/docs/Web/HTTP/Status/400

Http errors are fairly open to interpretation though, so there may be a better option.